Gemfile

```diff
@@ -2,4 +2,8 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gemspec :name => "logstash-core"
+gem "logstash-core", "2.0.0.dev", :path => "."
+gem "ci_reporter", "1.9.3", :group => :development
+gem "simplecov", :group => :development
+gem "coveralls", :group => :development
+gem "rspec", "~> 2.14.0", :group => :development
```

Comment:
This has been usually loaded throw devutils till now, do we want to change that?